mitosheet/src/mito/types.tsx

```diff
@@ -828,6 +828,12 @@ export interface KeyboardShorcut {
     // the command registered to that ID in plugin.tsx.
     jupyterLabCommand?: string
 
+    /**
+     * Some keyboard shortcuts should only be triggered if the user is not in a text input. 
+     * For example. cmd+a
```

Comment:
a little more here -- cmd + a should select the entire text input, not data in the sheet.